### PR TITLE
Fix typo

### PR DESCRIPTION
**Pull request by cmcconomy**
Offsensives -> Offensives

### CODE REVIEW

**Comment by cmcconomy** [comment]
Failure is due to 503 returned from abuseipdb; seems silly to have a pull request fail because one of the sites is having a maintenance issue:

> AbuseIPDB
> Is Down For Maintenance
> We apologize for any inconvenience.
> We are making changes to our site to improve your experience.
> 
> Please check back at a later time.

**Comment by ThunderSon** [comment]
No worries, the links if not related to the PR at hand are not blockers :)

File: cheatsheets/Abuse_Case_Cheat_Sheet.md [hunk 1]
```diff
@@ -101,7 +101,7 @@ Make a workshop that includes people with the following profiles:
...
-- **Offsensives (Pentester or Application Security people with offensive mindset)**: Will be the *attacker* that will propose all attacks that he can perform on the business feature(s) in question. If the company does not have a person with this profile then it is possible to request the service of an external specialist (Pentester or AppSec consultant from a security firm). If possible, include 2 offensives people (ex: 1 Pentester + 1 AppSec) in order to increase the number of possible attacks that will be identified and considered.
+- **Offensives (Pentester or Application Security people with offensive mindset)**: Will be the *attacker* that will propose all attacks that he can perform on the business feature(s) in question. If the company does not have a person with this profile then it is possible to request the service of an external specialist (Pentester or AppSec consultant from a security firm). If possible, include 2 offensives people (ex: 1 Pentester + 1 AppSec) in order to increase the number of possible attacks that will be identified and considered.
```
Comment (ThunderSon) [inline cheatsheets/Abuse_Case_Cheat_Sheet.md:104]: What do you think about this?
```suggestion
- **Security officer**: Will be the *attacker* that will propose attacks that they can perform on the business feature(s) in question. If the company does not have a person with this profile then it is possible to request the service of an external specialist. If possible, include 2 security officers with different backgrounds in order to increase the number of possible attacks that will be identified and considered.
```

Comment (cmcconomy) [reply cheatsheets/Abuse_Case_Cheat_Sheet.md:104]: I think it's an improvement.
Officer? Engineer?

Comment (rbsec) [reply cheatsheets/Abuse_Case_Cheat_Sheet.md:104]: If you're talking about an attacker, then "penetration tester" or "red team" would make more sense to me.

"security officer" is a bit vague - I don't really know what to expect from someone in that role, but I would think it's more on the defensive and governance side.

Comment (cmcconomy) [reply cheatsheets/Abuse_Case_Cheat_Sheet.md:104]: Good point - the role here is really limited to Attacker. Whether an organization's "Security Officer" fulfills this role as part of a broader mandate is outside the scope of this recommended approach, where targeted/specific roles are required to accomplish the activities.

This role must be capable of brainstorming potential feasible attacks on the application.

Comment (ThunderSon) [reply cheatsheets/Abuse_Case_Cheat_Sheet.md:104]: Please go forward with using "Penetration Tester"

Comment (cmcconomy) [reply cheatsheets/Abuse_Case_Cheat_Sheet.md:104]: Sorry to be a bother; I submitted this change directly via the github web ui - I'm unfamilar with how to revise the merge request. Could you let me know how best to revise my change? Should I cancel and resubmit or is there a way to add a subsequent revision directly on the site without checking out the repo?

Comment (mackowski) [reply cheatsheets/Abuse_Case_Cheat_Sheet.md:104]: You can go to the „Files changed” in this PR, click on three dots in the upper right corner and choose edit file.

Comment (mackowski) [reply cheatsheets/Abuse_Case_Cheat_Sheet.md:104]: Or this link should also work: https://github.com/cmcconomy/CheatSheetSeries/edit/patch-1/cheatsheets/Abuse_Case_Cheat_Sheet.md?pr=%2FOWASP%2FCheatSheetSeries%2Fpull%2F437

Comment (cmcconomy) [reply cheatsheets/Abuse_Case_Cheat_Sheet.md]: Thank you; second set of changes submitted.

File: cheatsheets/Abuse_Case_Cheat_Sheet.md [hunk 2]
```diff
@@ -101,7 +101,7 @@ Make a workshop that includes people with the following profiles:
...
-- **Offsensives (Pentester or Application Security people with offensive mindset)**: Will be the *attacker* that will propose all attacks that he can perform on the business feature(s) in question. If the company does not have a person with this profile then it is possible to request the service of an external specialist (Pentester or AppSec consultant from a security firm). If possible, include 2 offensives people (ex: 1 Pentester + 1 AppSec) in order to increase the number of possible attacks that will be identified and considered.
+- **Penetration Tester**: Will be the *attacker* that will propose all attacks that he can perform on the business feature(s) in question. If the company does not have a person with this profile then it is possible to request the service of an external specialist (Pentester or AppSec consultant from a security firm). If possible, include 2 Penetration Testers from diverse backgrounds in order to increase the number of possible attacks that will be identified and considered.
```
Comment (ThunderSon) [inline cheatsheets/Abuse_Case_Cheat_Sheet.md:104]:
```suggestion
- **Penetration Tester**: Will be the *attacker* that will propose attacks that they can perform on the business feature(s) in question. If the company does not have a person with this profile then it is possible to request the service of an external specialist. If possible, include 2 penetration testers with different backgrounds in order to increase the number of possible attacks that will be identified and considered.
```

File: cheatsheets/Abuse_Case_Cheat_Sheet.md [hunk 3]
```diff
@@ -176,17 +176,17 @@ Use the spreadsheet to review all the features.
 For each feature, follow this flow:
 
 1. Key business people explain the current feature from a business point of view.
-2. Offensives propose and explain a set of attacks that they can perform against the feature.
+2. Penetration Testers propose and explain a set of attacks that they can perform against the feature.
 3. For each attack proposed:
     1. Appsec proposes a countermeasure and a preferred set up location (infrastructure, network, code, design...).
```
Comment (ThunderSon) [inline cheatsheets/Abuse_Case_Cheat_Sheet.md:181]: this needs to be the pentester as well

**Comment by ThunderSon** [comment]
Thanks @cmcconomy !

**Pull request merged**